bionic/cache_api.py

```diff
@@ -18,13 +18,13 @@ class Cache:
     object. Use ``get_entries`` to iterate through the set of cache entries.
     """
 
-    def __init__(self, deriver):
+    def __init__(self, deriver_func):
```

Comment:
Rather than changing this interface, it might be easier to just re-create the Cache object when reloading happens.